gdcdictionary/schemas/treatment.yaml

```diff
@@ -68,6 +69,29 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  clinical_trial_indicator:
+    $ref:
+      - "_terms.yaml#/clinical_trial_indicator/common"
+    enum:
+      - "Yes"
+      - "No"
+      - Not Reported
```

Comment:
jira ticket says this should be Unknown?

Reply:
Changed ticket to "Not Reported" per @tlicht3